lib/common.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+
+//  formatErrors :: Array String -> String
+exports.formatErrors = function(errors) {
+  return (errors.map (function(s) { return 'error: ' + s + '\n'; })).join ('');
+};
+
+//  sanitizeFileContents :: String -> String
+exports.sanitizeFileContents = function(contents) {
+  return contents.replace (/\r\n?/g, '\n').replace (/^#!.*/, '');
+};
+
+//  unlines :: Array String -> String
+exports.unlines = function(lines) {
+  return lines.reduce (function(s, line) { return s + line + '\n'; }, '');
+};
```

Comment:
I realize that `common.js` might be a badly chosen name, seeing as it already means something within the very domain we're working with. I intended it to have "common functionality", which is where it gets the name.